src/topp/OpenSwathAssayGenerator.cpp

```diff
@@ -188,7 +230,7 @@ class TOPPOpenSwathAssayGenerator :
     }
 
     // Load Unimod file
-    if (enable_ms1_uis_scoring || enable_ms2_uis_scoring)
+    if (enable_ipf)
```

Comment:
I wonder whether it may make sense to control the unimod file even when not using IPF - wouldnt it give you more control over how OpenMS interprets the mods? So only allowing a custom unimod file when IPF is active maybe too restrictive, why not decouple the two options?

Reply:
Yes, I think that is a good idea, especially when importing data from SpectraST or generic transition lists that use bracket annotation. I will provide a patch to enable this option.